src/query/functions/src/scalars/geometry.rs

```diff
@@ -46,20 +51,19 @@ use geozero::ToWkt;
 pub fn register(registry: &mut FunctionRegistry) {
     // aliases
     registry.register_aliases("st_makegeompoint", &["st_geom_point"]);
+    registry.register_aliases("st_makepolygon", &["st_polygon"]);
     registry.register_aliases("st_makeline", &["st_make_line"]);
     registry.register_aliases("st_geometryfromwkb", &[
         "st_geomfromwkb",
         "st_geometryfromewkb",
         "st_geomfromewkb",
-        "to_geometry",
     ]);
     registry.register_aliases("st_geometryfromwkt", &[
         "st_geomfromwkt",
         "st_geometryfromewkt",
         "st_geomfromewkt",
         "st_geometryfromtext",
         "st_geomfromtext",
-        "to_geometry",
```

Comment:
Can we keep this alias function? we need auto cast from string type values to geometry type.

Reply:
https://github.com/datafuselabs/databend/discussions/15192
Because when user call to_geometry('0101000020797F000066666666A9CB17411F85EBC19E325641'), 
ST_GEOMETRYFROMWKT( <varchar_expression> ) will be called.But in ST_GEOMETRYFROMWKT, ST_GEOMETRYFROMWKB logic will execution.The logic of this code is a bit strange.

```
  pub fn register_aliases(&mut self, fn_name: &str, aliases: &[&str]) {
        for alias in aliases {
            self.aliases.insert(alias.to_string(), fn_name.to_string());
        }
    }
    registry.register_aliases("st_geometryfromwkb", &[
        "to_geometry",
    ]);
    registry.register_aliases("st_geometryfromwkt", &[
        "to_geometry",
    ]);
 registry.register_passthrough_nullable_1_arg::<StringType, GeometryType, _, _>(
        "st_geometryfromwkt",
        |_, _| FunctionDomain::MayThrow,
        vectorize_with_builder_1_arg::<StringType, GeometryType>(|wkt, builder, ctx| {
        match parse_to_subtype(binary) {
            GeometryDataType::EWKB => {
                 // call st_geometryfromwkb logic
            }),    
            GeometryDataType::EWKB => {
               // call st_geometryfromwkt logic
            }
         ......
);

```

I think it's better to implement it as independent function, not as an alias. 
```
 registry.register_passthrough_nullable_1_arg::<StringType, GeometryType, _, _>(
        "to_geometry",
 registry.register_passthrough_nullable_1_arg::<binary, GeometryType, _, _>(
        "to_geometry",
```

Let the community developers decide on the final implementation approach.

Reply:
ok, we can add this function later.